README.md

```diff
@@ -49,8 +49,8 @@ You first need to create a Python class that inherits from the SDK's `panoply.Da
 import panoply
 
 class MyDataSource(panoply.DataSource):
-  def __init__(self, source, options):
-    super(MyDataSource, self).__init__(source, options)
+  def __init__(self, *args, **kwargs):
+    super(MyDataSource, self).__init__(*args, **kwargs)
```

Comment:
???
where was that changed? and why?
i probably missed something but looks like signature is still `def __init__(self, source, options={}, events={})`